docs/messaging/gleanplumb.mdx

```diff
@@ -4,28 +4,28 @@ title: Mobile Messaging
 slug: /mobile-messaging
 ---
 
+# Introduction
+The mobile messaging system is a feature of Firefox on iOS and Android, designed to send in-app messages directly to users without going through a release cycle.
+
+It allows staff— most likely experiment owners, product owners, user research and marketing teams—
+
+ - to send messages to the user audiences, with rules about when to show them.
+ - to experiment with those messages (the appearance or copy of the message)
+ - to allow the user to respond to the message (either dismissing, or opening a specified URL)
+
```

Comment:
This is incidental to the patch, but I've done this to:

- remove reference to the Gleanplumb name
- make this introduction show instead of the links (below) show when previewing in slack.